mod/groups/start.php

```diff
@@ -15,6 +15,8 @@
  */
 function groups_init() {
 
+	elgg_register_plugin_hook_handler('route', 'groups', 'groups_router');
```

Comment:
Instead of this, we would add a route declaration:
```php
elgg_register_route('groups/all', \Elgg\Groups\Routes\Index::class);
elgg_register_route('groups/profile/{guid}', \Elgg\Groups\Routes\Profile::class);
```